addons/radar/XEH_preInit.sqf

```diff
@@ -19,6 +19,12 @@ GVAR(setNamelist) = true;
 // compass whitelist, these need to be lowercase!!
 GVAR(compassWhitelist) = "getText (_x >> 'simulation') == 'ItemCompass'" configClasses (configFile >> "CfgWeapons") apply {configName _x};
 
+// GPS whitelist
+private _uavTerminals = "inheritsFrom _x == (configFile >> 'CfgWeapons' >> 'UavTerminal_base')" configClasses (configFile >> "CfgWeapons") apply {configName _x};
+private _gpsDevices = "getText (_x >> 'simulation') == 'ItemGPS'" configClasses (configFile >> "CfgWeapons") apply {configName _x};
+
+GVAR(gpsWhitelist) = _uavTerminals + _gpsDevices;
```

Comment:
i would say instead of forcefully overwriting i would check if the variable was already defined and than not overwrite it so that the user/missionbuilder can define for them self what Device is counted as a GPS

Reply:
TBH, not much of an issue. Mission makers just need to define them during init or post init.